src/main/java/uk/gov/pay/directdebit/DirectDebitConnectorApp.java

```diff
@@ -21,12 +21,7 @@
 import uk.gov.pay.directdebit.app.config.GraphiteConfig;
 import uk.gov.pay.directdebit.app.healthcheck.Database;
 import uk.gov.pay.directdebit.app.healthcheck.Ping;
-import uk.gov.pay.directdebit.common.exception.BadRequestExceptionMapper;
-import uk.gov.pay.directdebit.common.exception.ConflictExceptionMapper;
-import uk.gov.pay.directdebit.common.exception.InternalServerErrorExceptionMapper;
-import uk.gov.pay.directdebit.common.exception.JsonMappingExceptionMapper;
-import uk.gov.pay.directdebit.common.exception.NotFoundExceptionMapper;
-import uk.gov.pay.directdebit.common.exception.PreconditionFailedExceptionMapper;
+import uk.gov.pay.directdebit.common.exception.*;
```

Comment:
We don't like wildcard imports, you probably need to change your intelliJ settings then re-run optimise imports.

https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports